gsr_booking/models.py

```diff
@@ -59,13 +59,27 @@ class Group(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    ADMIN = "A"
+    MEMBER = "M"
+
     def __str__(self):
         return "{}: {}".format(self.pk, self.name)
 
     def has_member(self, user):
         memberships = GroupMembership.objects.filter(group=self, accepted=True)
         return memberships.all().filter(user=user).exists()
 
+    def is_admin(self, user):
```

Comment:
maybe calling this `has_admin` to match `has_member` above? at first it looked like this was a method on GroupMembership, not a Group.